pyproject.toml

```
@@ -26,7 +26,7 @@ classifiers = [
 ]
 requires-python = ">= 3.11, < 4"
 dependencies = [
-  "frequenz-api-common >= 0.5.4, < 0.6",
+  "frequenz-api-common >= 0.6.0, < 0.7",
```

If this works with both v0.5.4 we should widen the dependency instead.
```suggestion
  "frequenz-api-common >= 0.5.4, < 0.7",
```